### DISCUSSION TITLE
New User

### DISCUSSION THREAD

**Discussion by Yardithbey** [ezra-bible-app/ezra-bible-app#1071] [General]
Greetings and thank you for your work on this app!
I am searching for a new Bible app since the one I've used for the past 20+ years is no longer supported and I therefore can't move it to my new computer.  I am trying out Ezra and have 3 comments / questions.
1. THANK YOU! Thank you for dark mode.  My tired eyes rejoice.
2. Is it possible to have 2 Bible windows (tabs?) open side by side.  Bonus if they scroll together.  I am used to having my English translation open next to my Greek/Hebrew.  If that is possible, I cannot figure it out. 
3. Does the app store any of the translations, books, etc. locally so I can use the app when off-line?

**Comment by tobias-klein** [comment]
@Yardithbey 
Thank you for your feedback! I am glad this app is useful for you.

> 2. Is it possible to have 2 Bible windows (tabs?) open side by side.  Bonus if they scroll together.  I am used to having my English translation open next to my Greek/Hebrew.  If that is possible, I cannot figure it out.

This is not implemented, yet. There is a feature request for this since a long time (#30).
I may use your feedback to investigate how quickly this could be added.
My feeling is that it does require a significant implementation effort.
However, I agree that this could be a very useful feature.

> 3. Does the app store any of the translations, books, etc. locally so I can use the app when off-line?

You can use the app offline.
All the Bible modules, commentaries, dictionaries are stored offline.
Your tags and notes are also stored offline.
